docs/Earn/liquid-staking/dot/stake-dot.md

```diff
@@ -47,18 +47,39 @@ You can use _Polka Wallet, Trust Wallet, Fearless Wallet, Polkadot.js_ to send K
 
 3. You can then view your balance and/or make a transfer if required.
 
-## Connect your Wallet
+## Connect your wallet
 
-1. Visit  [**Ankr Earn**](https://stakefi.ankr.com/) and click the *'Launchpad'* button.
+1. Visit [Ankr Earn](https://www.ankr.com/earn/) and click **Stake**.
 
-2. Locate the *‘Stake DOT* box and click *‘Start staking’*.
+2. Locate the **Stake DOT** box and click *Stake* in it.
 
 3. Your ```Polkadot.js wallet``` pops up requesting **'authorization'** to connect to Ankr Earn.
 Click to allow access.
 
-## Stake Desired Amount
+## Stake your DOT
 
-1. The **DOT Staking Dashboard** opens
+1. Enter the desired amount of AVAX to stake. If you already have some staked AVAX and want to add more, click the **'+'** button.
+
+2. Your wallet pops with details of the transaction including a suggested gas fee.
+
+3. Click ‘**Confirm**’ to proceed.
+
+4. The status of the transaction updates in your wallet.
+
+5. When staking is finalized, the **Staking Dashboard** updates.
+
+
+1. Enter the desired amount of AVAX to stake. If you already have some staked AVAX and want to add more, click the **'+'** button.
+
+2. Your wallet pops with details of the transaction including a suggested gas fee.
+
+3. Click ‘**Confirm**’ to proceed.
+
+4. The status of the transaction updates in your wallet.
+
+5. When staking is finalized, the **Staking Dashboard** updates.
+
+1. The **DOT Staking Dashboard** opens.
 
 2. To stake, click the **'+'** icon.
 
```

Comment:
Is this a repeated section on the same page or weird numbering?

Reply:
fixed, thx